app/models/team.rb

```diff
@@ -194,7 +194,11 @@ def self.import(row, id, options, teamtype)
     end
 
     if name
-      team = Team.create_team_and_node(id, teamtype)
+      if teamtype.is_a?(CourseTeam)
+        team = CourseTeam.create_team_and_node(id)
```

Comment:
W8, there is no 'create_team_and_node' defined separately in AssignmentTeam model and CourseTeam model. What's the point of this change, then?